conf/routes

```diff
@@ -61,7 +61,6 @@ GET  /downloadCSV      com.lynxanalytics.biggraph.serving.ProductionJsonServer.d
 
 GET  /getLogFiles      com.lynxanalytics.biggraph.serving.ProductionJsonServer.getLogFiles
 GET  /downloadLogFile  com.lynxanalytics.biggraph.serving.ProductionJsonServer.downloadLogFile
-POST /forceLogRotate   com.lynxanalytics.biggraph.serving.ProductionJsonServer.forceLogRotate
```

Comment:
Our log setup was chaos anyway. We hereby leave it all for the environment to manage.